include/bluetooth/bluetooth.h

```diff
@@ -960,6 +960,9 @@ struct bt_le_per_adv_sync_synced_info {
 
 	/** Advertiser PHY */
 	uint8_t phy;
+
+	/** True if not receiving periodic advertisements, false otherwise. */
+	bool recv_disabled;
```

Comment:
recv_enabled would be better IMO.

Reply:
I actually had that in the beginning, but changed it around. The reason why I decided on the disable rather than enable, is that it is enabled by default from the controller. Disable is the state that requires extra attention, and thus the one that should perhaps be captured by the variable. 

That being said, I do actually like enabled better, so I'll switch it around :)